### PR TITLE
Pop up Window

### PR DESCRIPTION
**Pull request by mwanawabangona**
- Add Html
- Add Css
- Add JavaScript
- Add Linters

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  year.innerHTML = project.year;
+  descrition.innerHTML = project.decription;
+  const tagsLength = project.tags.length;
+  for (let i = 0; i < tagsLength; i += 1) {
+    const element = document.createElement('li');
+    element.innerHTML = `${project.tags[i]}`;
+    tags.appendChild(element);
+  }
+  openPopUp();
+}
```
Comment (DanteAlonsoHT) [inline main.js]: - [x] Add responsive design for your popup, in the image below you can see how it looks like in a mobile version:
- [x] Add more padding for your main content which contain all your information in your popup window, you can see that it is so close (check the yellow color in the image below)
- [x] [OPTIONAL] Add the next CSS attribute to improve the UX in your app:
```
button:hover {
  cursor: pointer;
}
```

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @mwanawabangona, 

Good job so far, I really love your design and the way you use to make the popup window! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
### _Check the comments under the review._ 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (mwanawabangona) [reply main.js]: done, thank you

Comment (rloterh) [reply main.js]: This aspect needs further improvements.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+});
+
+popupOverlay.addEventListener('click', closePopUp);
+btnClosePop.addEventListener('click', closePopUp);
+
+const form = document.getElementById('contact-form');
+const email = document.getElementById('email');
+const msg = document.querySelector('.show-msg');
+
+function validationEmail(e) {
```
Comment (rloterh) [inline main.js]: - [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

**Comment by mwanawabangona** [comment]
what improvements, should implement

File: main.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  job.innerHTML = project.job;
+  year.innerHTML = project.year;
+  descrition.innerHTML = project.decription;
+  const tagsLength = project.tags.length;
+  for (let i = 0; i < tagsLength; i += 1) {
+    const element = document.createElement('li');
+    element.innerHTML = `${project.tags[i]}`;
+    tags.appendChild(element);
+  }
+  openPopUp();
```
Comment (rloterh) [inline main.js]: - [ ] You did a great job by adding paddings around the contents of the _popup window_ for the Desktop screens. However, the  _popup window_  for mobile screens needs some spacing around the contents, as can be seen in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Kindly endeavor to improve upon this aspect.

**Your pop-up window (mobile screen):**
image



**The original pop-up window (mobile screen):**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Ellis, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments above, made below /above the reviews of the previous code reviewer(s)**. Kindly take your time to correctly review those issues outlined by the previous code reviewers, and please check to see that the changes are working as intended._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider the comments marked **[OPTIONAL]** under this review:

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 3 above)
Comment (mwanawabangona) [reply main.js]: Hi, this works just fine on my browser, I don't why it is showing you like that, if possible I would like to schedule a zoom call so I show you

**Pull request closed without merging**